cypress/support/step_definitions/index.ts

```diff
@@ -116,6 +147,15 @@ Then(/^the "(.*)" element( does| doesn't| don't)? exist[s]?$/, function (selecto
 Then(/^the "(.*)" element[s]? exist[s]? ([0-9]*) time[s]?$/, (selector: string, count: string) => {
   $(selector).should('have.length', count)
 })
+
+Then(/^the "(.*)" element[s]? (?:is|are) (enabled|disabled|checked|not checked)$/, (selector: string, word: string) => {
+  const prefix = ['enabled', 'not checked'].includes(word) ? 'not.' : ''
+  const property = ['disabled', 'enabled'].includes(word) ? 'disabled' : 'checked'
+  const chainer = `${prefix}be.${property}`
+
+  $(selector).should(chainer)
+})
+
```

Comment:
I’ve generalized this a bit now covering some likely word cases as suggested.

Reply:
I'm curious as to why:

1. We can't use `n't` like other steps and which is how people usually talk/write.
2. We can't use `${prefix}be.${word}`

Also, am I reading this wrong or can I say `the "selector" element is "writeable"` and it translates that to `be.readonly`, still looking at this but I'm a bit 🤔  Would it be easier implement points 1 and 2 here ^?

Reply:
> We can't use n't like other steps and which is how people usually talk/write.

We can, yes, but the possibility tree becomes significantly larger.

I guess your intention is actually to only allow actual attribute names for `word` (e.g. `disabled` but not `enabled`), is that correct?

> Also, am I reading this wrong or can I say the "selector" element is "writeable" and it translates that to be.readonly, 

That would be `'not.be.readonly'`.

Reply:
> That would be 'not.be.readonly'.

Ah yeah read it wrong 😆 

> I guess your intention is actually to only allow actual attribute names for word (e.g. disabled but not enabled), is that correct?

I think we should allow anything, ie "does the step have `n't`? then add `.not`, other wise just pass through the property". In the positive sense it would fail if it wasn't there, granted in the negative it wouldn't I suppose, but I'm kinda trying to avoid adding a big list of strings that will never be complete here just to check against. If you are super keen on having a big list of stuff then happy to leave this one aside.

I think my main thing is to try and keep the steps consistent, we use `n't` elsewhere so I'm like "can we keep this the same?", I'm strongest on that really as anything else we can change as we go. Is the `n't` absolute no-go?

Reply:
I’ve simplified the two utilities around these assertions and made them support the same style of assertions now.